main/src/test/resources/org/apache/james/jdkim/corpus/multiHeader.eml

```diff
@@ -0,0 +1,40 @@
+Received: from v4.mailinblack.org ([192.168.160.165])
+        by mib.mailinblack.com
+        with SMTP (SubEthaSMTP null) id J2VY6B21
+        for cballe@mailinblack.com;
+        Fri, 19 May 2017 16:35:35 +0200 (CEST)
+Received: from letsignit.sqli.com (letsignit.sqli.com [193.105.238.150])
+        (using TLSv1 with cipher ADH-AES256-SHA (256/256 bits))
+        (No client certificate requested)
+        by v4.mailinblack.org (Postfix) with ESMTPS id 26C07AC033
+        for <cballe@mailinblack.com>; Fri, 19 May 2017 16:35:35 +0200 (CEST)
+Received: from letsignit.sqli.com (localhost [127.0.0.1])
+        by letsignit.sqli.com (Postfix) with ESMTP id C107D580108
+        for <cballe@mailinblack.com>; Fri, 19 May 2017 16:35:34 +0200 (CEST)
+DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=sqli.com; s=mail;
+        t=1495204534; bh=7rIC1gdWBQp5oiCoqvFeWrl/plYm+72BA6KkoHkY0Y4=;
+        h=Date:To:From:subject:Subject:From;
+        b=fwEo9e/VyCdDggbuhLRBEbaCwcIx4x2GHlIpNA2phygpku50i86NnwBZhPNFAqAmI
+         GVA/JjW73Ealf3bANzKJl3FyHNAyThXfHzMZWYL6I9u+T5h8iFx07FvBaSyDMGtrW7
+         IBxEhFxzNwXcaBVMy/73N/VxPj+uZSy21MvzKWus=
+Received: from letsignit.sqli.com (localhost [127.0.0.1])
+        by localhost
+        with SMTP (LetSignIt ESMTP MAIL Service, Version: 1.0 ready at Fri May 19 16:35:34 CEST 2017) id A088A8AED87240C5BAF16CE0985D91EF
+        for cballe@mailinblack.com;
+        Fri, 19 May 2017 16:35:34 +0200 (CEST)
+Received: from letsignit.sqli.com (localhost [127.0.0.1])
+        by letsignit.sqli.com (Postfix) with ESMTP id B721D580108
+        for <cballe@mailinblack.com>; Fri, 19 May 2017 16:35:34 +0200 (CEST)
+inLsi: true
+inLsi: true
+Date: Thu, 18 May 2017 05:30:06 +0200
+To: support+warning@letsignit.com,cballe@mailinblack.com
```

Comment:
Provided with the contribution...

Also, signature depends on these headers, changing them won't be easy...